integration/templates/coredns_values.go

```diff
@@ -3,12 +3,14 @@
 package templates
 
 // CoreDNSValues values required by coredns-app.
-const CoreDNSValues = `namespace: kube-system
+const CoreDNSValues = `namespace: default
+
+name: coredns-test
 
 cluster:
   kubernetes:
     API:
       clusterIPRange: 10.96.0.0/16
     DNS:
-      IP: 10.96.0.10
+      IP: 10.96.0.11
```

Comment:
Why did this change?

Reply:
Maybe regular coredns is `.10`? But Eric please confirm

Reply:
@stone-z yep that is correct. If you don't change the coredns-app static svc ip it will cause an ip conflict with the kind's coredns svc.


This is from a vanilla kind cluster:
```
k -n kube-system get svc -o wide
kind-test
NAME       TYPE        CLUSTER-IP   EXTERNAL-IP   PORT(S)                  AGE   SELECTOR
kube-dns   ClusterIP   10.96.0.10   <none>        53/UDP,53/TCP,9153/TCP   70s   k8s-app=kube-dns
```